go/chat/context.go

```diff
@@ -93,7 +89,7 @@ func CtxTrace(ctx context.Context) (string, bool) {
 	return "", false
 }
 
-func CtxAddLogTags(ctx context.Context, env appTypeSource) context.Context {
+func CtxAddLogTags(ctx context.Context, g *globals.Context) context.Context {
```

Comment:
Could we rollback this change too?

Reply:
I think this will break the mobile app compiling.

Reply:
done